beacon-chain/rpc/attester_server.go

```diff
@@ -37,11 +37,12 @@ func (as *AttesterServer) SubmitAttestation(ctx context.Context, att *ethpb.Atte
 		return nil, errors.Wrap(err, "failed to sign root attestation")
 	}
 
-	if err := as.operationsHandler.HandleAttestation(ctx, att); err != nil {
-		return nil, err
-	}
-
 	go func() {
+		ctx = context.Background()
```

Comment:
This is going to break the tracing. Can you inherit the span from the parent context? 

Something like 
```go
ctx := tracing.ContextWithSpan(tracing.SpanFromContext(ctx), context.Background())
```

Reply:
Alright changed it